docs/configuration.md

```diff
@@ -1,8 +1,12 @@
 # Configuration Reference
 
-All configuration should come from RancherOS minimal `cloud-init`.
-Below is a reference of supported configuration.  It is important
-that the config always starts with `#cloud-config`
+All custom configuration applied on top of a fresh deployment should come
+from a minimal `cloud-config` data. The `cloud-config` data can eventually
+be included within the OS image as a file in `/system/oem` or,
+alternatively, it can also be distributed from the Kubernetes management
+cluster as part of the machine registration data.
+
+Below is a reference of supported configuration. 
 
 ```yaml
```

Comment:
We should just have a decent example on the repo and call it from here to be included so its always updated

Reply:
Agree, however this is not that straight forward. The example should come from YIP, as this is where this data is going to be de-serialized. I don't think we have a cloud-config (aka the supported cloud-init syntax) reference in YIP. I think we only have a reference for it's own syntax.

Reply:
argh I thougth we had..that sucks, maybe we can send a patch to yip for that as it makes sense to have a reference.